src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -54,9 +54,12 @@ else
                                 <a class="btn btn-sm btn-primary" edit-for="@Model.ListPart.ContentItem" asp-route-returnUrl="@FullRequestPath">
                                     @T["Edit {0}", @Model.ListPart.ContentItem.DisplayText]
                                 </a>
-                                <a class="btn btn-sm btn-success" asp-action="Create" asp-controller="Admin" asp-route-id="@contentTypeDefinition.Name" asp-route-area="OrchardCore.Contents" asp-route-ListPart.ContainerId="@Model.ListPart.ContentItem.ContentItemId" asp-route-ListPart.ContentType="@contentTypeDefinition.Name" asp-route-ListPart.EnableOrdering="@Model.EnableOrdering" asp-route-returnUrl="@FullRequestPath">
-                                    @T["Create {0}", contentTypeDefinition.DisplayName]
-                                </a>
+                                @if (Model.IsContentCreatable)
+                                {
+                                    <a class="btn btn-sm btn-success" asp-action="Create" asp-controller="Admin" asp-route-id="@contentTypeDefinition.Name" asp-route-area="OrchardCore.Contents" asp-route-ListPart.ContainerId="@Model.ListPart.ContentItem.ContentItemId" asp-route-ListPart.ContentType="@contentTypeDefinition.Name" asp-route-ListPart.EnableOrdering="@Model.EnableOrdering" asp-route-returnUrl="@FullRequestPath">
```

Comment:
What @Piedone says. You will find on line 75, that when multiple types are available for the list, they are displayed differently, and need to be checked for permission as well.

I would suggest that it is actually easier to do the authorization check in the view, at the time of rendering, than doing it in the driver.

Reply:
I see that makes sense. Thank you.